sdk-setup/src/sdk-manage

```diff
@@ -651,6 +651,14 @@ object_zypper() {
     return $rc
 }
 
+which_sdk_register() {
+    local object=$1
+    local sdk_register=$(enter "$object" which sdk-register)
+    enter "$object" bash -c "[[ -x /usr/libexec/sdk-register ]]" && \
+        sdk_register=/usr/libexec/sdk-register
+    echo $sdk_register
```

Comment:
I was rather thinking about using /usr/libexec/sdk-setup/sdk-register as the location (not an issue that sdk-register lives in a separate package, provided that we do not use it on devices)?

Independently on that, I'd also suggest to do:

```suggestion
    enter "$object" bash -c 'PATH=/usr/libexec/sdk-setup:$PATH which sdk-register'
```

Reply:
> I was rather thinking about using /usr/libexec/sdk-setup/sdk-register as the location (not an issue that sdk-register lives in a separate package, provided that we do not use it on devices)?

Sure, I suppose that makes sense. sdk-setup and sdk-register are quite closely related.